### PR TITLE
feature/#5107/The visualization of the Event preview card and display of its elements is not matched with their design

### PR DESCRIPTION
**Pull request by SokhaTaras**
Page before: 

image

Page after:

image

Card before:
image

Card after:
image

### CODE REVIEW

**Review by vplvua — APPROVED** [review]

**Review by AngelinaRudnitska — APPROVED** [review]

File: src/app/main/component/events/components/events-list/events-list.component.scss [hunk 1]
```diff
@@ -46,6 +46,8 @@
 
   .event-list-item {
     overflow: hidden;
-    background-color: var(--primary-light-grey);
+    max-width: 359px;
+    background-color: #fff;
```
Comment (hnativlyubomyr) [inline src/app/main/component/events/components/events-list/events-list.component.scss]: use variable for color here

File: src/app/main/component/shared/components/events-list-item/events-list-item.component.scss [hunk 2]
```diff
@@ -3,26 +3,83 @@
...
+        font-weight: 500;
+      }
+    }
+  }
+
   .event-stars {
     width: auto;
     height: 40px;
-    background-color: rgba(224, 233, 233, 0.781);
+    background-color: #fffc;
```
Comment (hnativlyubomyr) [inline src/app/main/component/shared/components/events-list-item/events-list-item.component.scss]: the same

File: src/app/main/component/shared/components/events-list-item/events-list-item.component.scss [hunk 3]
```diff
@@ -48,12 +105,15 @@
   .event-flags {
     width: 40px;
     height: 40px;
-    background-color: rgba(224, 233, 233, 0.781);
+    background-color: #fff;
```
Comment (hnativlyubomyr) [inline src/app/main/component/shared/components/events-list-item/events-list-item.component.scss]: the same

File: src/app/main/component/shared/components/events-list-item/events-list-item.component.ts [hunk 4]
```diff
@@ -87,15 +99,19 @@ export class EventsListItemComponent implements OnInit, OnDestroy {
 
   public filterTags(tags: Array<TagDto>) {
     this.itemTags.forEach((item) => (item.isActive = tags.some((name) => name.nameEn === item.nameEn)));
+
+    this.activeTags = this.itemTags.filter((val) => {
+      return val.isActive;
```
Comment (hnativlyubomyr) [inline src/app/main/component/shared/components/events-list-item/events-list-item.component.ts]: don't use return here
this.activeTags = this.itemTags.filter((val) =>val.isActive)

File: src/app/main/component/shared/components/events-list-item/events-list-item.component.ts [hunk 5]
```diff
@@ -238,6 +254,34 @@ export class EventsListItemComponent implements OnInit, OnDestroy {
...
+      this.newDate = this.datePipe.transform(new Date(), 'MMM dd, yyyy');
+    });
+  }
+
+  cutTitle(): string {
+    if (this.event.title.length > 40) {
+      return this.event.title.slice(0, 30) + '...';
+    } else {
+      return this.event.title;
+    }
```
Comment (hnativlyubomyr) [inline src/app/main/component/shared/components/events-list-item/events-list-item.component.ts]: use ternary operator here
return this.event.title.length > 40 ? this.event.title.slice(0, 30) + '...'  :  this.event.title

File: src/app/main/component/shared/components/events-list-item/events-list-item.component.ts [hunk 6]
```diff
@@ -238,6 +254,34 @@ export class EventsListItemComponent implements OnInit, OnDestroy {
...
+  }
+
+  cutDescription(): string {
+    if (this.event.description.length > 90) {
+      return this.event.description.slice(0, 90) + '...';
+    } else {
+      return this.event.description;
```
Comment (hnativlyubomyr) [inline src/app/main/component/shared/components/events-list-item/events-list-item.component.ts]: the same

**Review by hnativlyubomyr — APPROVED** [review]

**Review by atsa21 — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by marianaharnyk — APPROVED** [review]

**Pull request merged**